Chunks/chunk_queue.py

```diff
@@ -131,8 +132,19 @@ def create_video_queue_hybrid(args: Args, split_locations: List[int]) -> List[Ch
     return chunk_queue
 
 
-# TODO: Either update the name of this func, or change the docs, or create a new func just for vapoursynth
 def create_video_queue_vsffms2(args: Args, split_locations: List[int]) -> List[Chunk]:
+    script = "from vapoursynth import core\n" \
+             "core.ffms2.Source(\"{}\", cachefile=\"{}\").set_output()"
+    return create_video_queue_vs(args, split_locations, script)
+
+
+def create_video_queue_vslsmash(args: Args, split_locations: List[int]) -> List[Chunk]:
+    script = "from vapoursynth import core\n" \
+             "core.lsmas.LWLibavSource(\"{}\", cachefile=\"{}\").set_output()"
+    return create_video_queue_vs(args, split_locations, script)
+
+
+def create_video_queue_vs(args: Args, split_locations: List[int], script: List[str]) -> List[Chunk]:
```

Comment:
```suggestion
def create_video_queue_vs(args: Args, split_locations: List[int], script: str) -> List[Chunk]:
```

Can you also add the script param to the docstring for this function?